samples/utility/Backup-VMAttributes.ps1

```diff
@@ -0,0 +1,91 @@
+<#
```

Comment:
Should be in protectionjob folder as well, and also remove the vcenterIPs, put a fqdn. This and script below.

Reply:
The scripts do not use cohesity cmdlet, does not make sense to place it inside protectionJob folder